system/Config/View.php

```diff
@@ -16,6 +16,14 @@
  */
 class View extends BaseConfig
 {
+	/**
+	 * When false, the view method will clear the data between each
+	 * call.
+	 *
+	 * @var boolean
+	 */
+	public $saveData = true;
```

Comment:
it's can be `false` by default - `RendererInterface::render(string $view, array $options = null, bool $saveData = false)`

Reply:
It is `true` in **app/Config/View.php** which extends this, so I think it makes more sense to mirror that.

Reply:
@MGatner why not same to `RendererInterface::render()`?

Reply:
I have no clue, I didn't write any of this. Probably a mistake or miscommunication among earlier devs. At this point though we have a public interface which we can't change and an App config file which we could change but would need to note in docs - I think keeping the config files consistent even though they don't match the interface makes the most sense given the mess it already is.